addons/account_asset/account_asset.py

```diff
@@ -115,8 +116,14 @@ def _get_last_depreciation_date(self):
 
     @api.model
     def _cron_generate_entries(self):
-        assets = self.env['account.asset.asset'].search([('state', '=', 'open')])
-        assets._compute_entries(datetime.today())
+
```

Comment:
I'd create an extra function to do:
```
    def compute_all_assets(self):
        # Entries generated : one by grouped category and one by asset from ungrouped category
         assets = self.env['account.asset.asset'].search([('category_id.group_entries', '=', False), ('state', '=', 'open')])
         assets._compute_entries(datetime.today(), group_entries=False)
 
         for category in self.env['account.asset.category'].search([('group_entries', '=', True)]):
             assets = self.env['account.asset.asset'].search([('category_id', '=', category.id), ('state', '=', 'open')])
             assets._compute_entries(datetime.today(), group_entries=True)
```
so that you just have to call that function in the cron and wizard, without having to duplicate code